backbone.queryparams.js

```diff
@@ -360,7 +360,11 @@ function iterateQueryString(queryString, callback) {
 }
 
 function containsInvalidKey(keys) {
-  return INVALID_KEYS.some(invalidKey => keys.includes(invalidKey));
+  // Square brackets will be removed down the line so
+  // we want to make sure they are filtered upfront.
+  const regex = /(\[|\])+/gi;
```

Comment:
Not sure if project that depends on this library still supports legacy browsers.
But maybe good to use `var` consistently, just in case.

Reply:
Good call, I have also replaced the INVALID_KEYS to use var instead.